src/nvim/ex_docmd.c

```diff
@@ -3275,7 +3275,7 @@ void ex_ni(exarg_T *eap)
 {
   if (!eap->skip)
     eap->errmsg = (char_u *)N_(
-        "E319: Sorry, the command is not available in this version");
+        "E319: The command is not available in this version");
```

Comment:
@Pyrohh Every time the English message changes, *all* translations are impacted. They might have become slightly off, or completely wrong. And nobody is notified that the translation needs to be updated. That's a quality issue.

Perhaps we will address this in #2150, but it's certainly something to be aware of.